### ISSUE TITLE
pymnn如何实现每个batch内输入图像大小一样，每个batch之间输入图像大小不一样

### ISSUE THREAD

**Issue by Danee-wawawa** [alibaba/MNN#959]
我想问下利用pymnn代码如何实现让每个batch内输入图像大小一样，每个batch之间输入图像大小不一样，pymnn提供的https://github.com/alibaba/MNN/blob/master/pymnn/examples/MNNTrain/quantization_aware_training/quant_aware_training.py
这个函数中主要是以下语句实现DataLoader：

train_dataset = ImagenetDataset(image_path, val_txt, True)
test_dataset = ImagenetDataset(image_path, val_txt, False)
train_dataloader = MNN.data.DataLoader(train_dataset, batch_size=32, shuffle=True)
test_dataloader = MNN.data.DataLoader(test_dataset, batch_size=10, shuffle=False)

我看了下ImagenetDataset以及MNN.data.DataLoader，它里面没有类似Pytorch的torch.utils.data.DataLoader中的collate_fn这样的参数以实现上述功能。那该如何实现呢？麻烦指导下。谢谢。
